src/platform/storage/zephyr/nvs/storage.c

```diff
@@ -52,6 +57,24 @@ static char *cached_artifact_name = NULL;
  */
 static struct nvs_fs mender_storage_nvs_handle;
 
+#ifdef CONFIG_MENDER_DEPLOYMENT_LOGS
+/**
+ * @brief A Flash Circular Buffer for storing deployment logs
+ */
+static struct fcb depl_logs_buffer;
+
+#define DEPL_LOGS_MAX_MSG_LEN 255
```

Comment:
Could we have this parameter configured by the user (Kconfig)? Just asking, good to know for future extensions.

Reply:
Yes, we could. My only concern is that then it could be different between different versions/images and that could cause a mismatch/misalignment. I guess we would then need to use the `version` parameter of the FCB a ensure there's some backwards compatibility. For now, I'd keep it fixed. That's also what I have seen in other logging backends I checked.